docs/sql-ref-syntax-aux-show-partitions.md

```diff
@@ -18,5 +18,96 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
 
-**This page is under construction**
+The `SHOW PARTITIONS` statement is used to list partitions of a table. An optional
+partition spec may be specified to return the partitions matching the supplied
+partition spec.
+
+### Syntax
+{% highlight sql %}
+SHOW PARTITIONS table_identifier [partition_spec]
+{% endhighlight %}
+
+### Parameters
+<dl>
+  <dt><code><em>table_identifier</em></code></dt>
```

Comment:
Shall we use `table_name` instead of `table_identifier` because it's used more.

```
$ git grep '<em>table_name</em>'
sql-ref-syntax-aux-analyze-table.md:  <dt><code><em>table_name</em></code></dt>
sql-ref-syntax-aux-cache-cache-table.md:  <dt><code><em>table_name</em></code></dt>
sql-ref-syntax-aux-cache-uncache-table.md: <dt><code><em>table_name</em></code></dt>
sql-ref-syntax-ddl-alter-table.md:  <dt><code><em>table_name</em></code></dt>
sql-ref-syntax-ddl-alter-table.md:  <dt><code><em>table_name</em></code></dt>
sql-ref-syntax-ddl-drop-table.md:  <dt><code><em>table_name</em></code></dt>
sql-ref-syntax-ddl-repair-table.md:  <dt><code><em>table_name</em></code></dt>
sql-ref-syntax-ddl-truncate-table.md:  <dt><code><em>table_name</em></code></dt>
sql-ref-syntax-dml-insert-into.md:  <dt><code><em>table_name</em></code></dt>
sql-ref-syntax-dml-insert-overwrite-table.md:  <dt><code><em>table_name</em></code></dt>
sql-ref-syntax-dml-load.md:  <dt><code><em>table_name</em></code></dt>

$ git grep '<em>table_identifier</em>'
sql-ref-syntax-aux-describe-table.md:  <dt><code><em>table_identifier</em></code></dt>
sql-ref-syntax-aux-show-tblproperties.md:  <dt><code><em>table_identifier</em></code></dt>
```

Reply:
Also, we used `table_name` at line 39 of this PR.

Reply:
@dongjoon-hyun For consistency we can. But personally, i would like to tell the users that we accept a table identifier in this command i.e it can be a qualified table. 

For example if we look at the following link.
https://docs.databricks.com/spark/latest/spark-sql/language-manual/cache-table.html

We do mention in above in the syntax that the table can be qualified. However, in our doc its not so clear. Thats the reason, i choose to define "table_identifier" as a parameter and had a sub syntax to explain further so there is no ambiguity. Another option is to just have [db_name].table_name in the main syntax block itself.
```
SHOW PARTITIONS [db_name].table_name 
   [ PARTITION ( partition_col_name = partition_col_val [ , ... ] ) ]
```

Please let me know what you think.

Reply:
If you think so, you are trying to do two things in this single PR. 

You had better follow the existing `table_name` in this PR first. Then, you can create another PR to adjust all of them from the docs. In general, we didn't agree your suggestion is good or bad. In worst case, you know that your suggestion can be rejected or we may choose another 3rd option.

Reply:
@dongjoon-hyun Sure.. I have made the change for this pr.